save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/demo/diktat/DemoAdditionalParams.kt

```diff
@@ -14,6 +14,109 @@ import kotlinx.serialization.Serializable
 data class DemoAdditionalParams(
     val mode: DiktatDemoMode = DiktatDemoMode.WARN,
     val tool: DiktatDemoTool = DiktatDemoTool.DIKTAT,
-    val config: String? = null,
-    val language: Languages = Languages.KOTLIN
-) : DemoAdditionalParams
+    val config: List<String> = defaultDiktatConfig,
+    val language: Languages = Languages.KOTLIN,
+) : DemoAdditionalParams {
+    companion object {
+        val defaultDiktatConfig = """
```

Comment:
maybe extract to a file in /resources?

Reply:
I think of adding it to storage, but in this case I will need to get it from storage if and only if there is no config provided. Will be done later in next PRs